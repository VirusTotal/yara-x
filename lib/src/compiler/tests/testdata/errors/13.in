rule test {
  strings:
    $a = "foo"
  condition: 
    $a in (-1..0)
}