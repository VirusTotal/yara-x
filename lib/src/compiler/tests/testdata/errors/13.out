error: unexpected negative number
 --> line:5:12
  |
5 |     $a in (-1..0)
  |            ^^ this number can not be negative
  |